MAKE/Makefile.lumi_cpeGnu

```diff
@@ -0,0 +1,90 @@
+CMP = CC
+LNK = CC
+
+# Modules loaded
+# module load PrgEnv-gnu
+# module load gcc/10.3.0
+# module load craype-x86-milan
+# module load papi
+
+## OR optionally instead of papi:
+# module load perftools
+
+#======== Vectorization ==========
+#Set vector backend type for vlasov solvers, sets precision and length. 
+#Options: 
+# AVX:	    VEC4D_AGNER, VEC4F_AGNER, VEC8F_AGNER
+# AVX512:   VEC8D_AGNER, VEC16F_AGNER
+# Fallback: VEC4D_FALLBACK, VEC4F_FALLBACK, VEC8F_FALLBACK
+
+ifeq ($(DISTRIBUTION_FP_PRECISION),SPF)
+#Single-precision        
+	VECTORCLASS = VEC8F_AGNER
+else
+#Double-precision
+	VECTORCLASS = VEC4D_AGNER
+endif
+
+FLAGS = 
+
+#GNU flags:
+CXXFLAGS += -g -ggdb -O3 -fopenmp -funroll-loops -std=c++17 -W -Wall -Wno-unused -mfma -mavx2 -Wno-unused-parameter
+testpackage: CXXFLAGS = -g -O2 -fopenmp -funroll-loops -std=c++20 -mno-fma -mno-avx2 -mno-avx
```

Comment:
Is -ggdb affecting performance? Should it be included for testpackage too? Why c++17 vs c++20?